### DISCUSSION TITLE
simple read/write code example for polylines and bulges?

### DISCUSSION THREAD

**Discussion by BennoPaul** [haplokuon/netDxf#316] [General]
netDxf is very impressive but also overwhelmingly comprehensive. For me as a newby it would be extremely helpful if I could get a simple short code skeleton which contains only the reading from a dxf files (vertices, lines, polylines, bulges), and writing the same after some changes. Could anyone help? Would be much appreciated!

**Comment by haplokuon** [comment]
In the TestDxfDocument project you can find all kind of samples on how to open, save, and edit the content of DXF files. It is better if you show what have you being doing and what type of problems you found along the way. Reading the information of a LwPolyline is as easy as
```c#
LwPolyline poly = doc.Entities.LwPolylines.ElementAt(index);
```
Then you can edit the properties of the desired vertexes and save the file again. 
```c#
LwPolylineVertex vertex = poly.Vertexes[index];
Vector2 position = vertex.Position;
double bulge = vertex.Bulge;
```
This process is very straightforward.
